doc/source/conf.py

```diff
@@ -35,21 +35,6 @@ def setup(app):
 # -- Referencable variables --------------------------------------------------
 
 
-def _get_latest_tag():
```

Comment:
This was used to set a variable, which in turn was never used. Let's instead rely on chartpress logic to extract something in a consistent manner.